functions/src/triggers/userCreate.ts

```diff
@@ -0,0 +1,21 @@
+import * as functions from 'firebase-functions'
+import * as jsonWebToken from 'jsonwebtoken'
+import { addTokenToUser } from '../db/addTokenToUser'
+
+export const userCreate = functions.auth.user().onCreate(async (user) => {
+    const { jwt } = functions.config()
+    if (!jwt.key) {
+        throw new functions.https.HttpsError(
+            'failed-precondition',
+            'Missing JWT Key'
+        )
+    }
```

Comment:
this lines could be moved in a `utility` file: pls consider getting the token through a utility function:

`const {jwt} = getJwtToken()`

This would eliminate duplication in `authentificateJWT.ts`